hycruz-entrypoint.sh

```diff
@@ -23,7 +23,7 @@ fi
 bundle check || bundle install
 
 #echo "Running Database Migration"
-bundle exec rake db:migrate 
+#bundle exec rake db:migrate 
```

Comment:
Commented out `bundle exec rake db:migrate`. I believe I need to understand better bashing into other folders from `src/docker`. How does one bash into Hyrax to run the following commands:

```
bundle exec rake db:migrate 
bundle exec rake db:seed 
bundle exec rake hyrax:workflow:load
bundle exec rake hyrax:default_admin_set:create
```

Commands that you ran like `docker exec -it hycruz bash` seemed to open a bash but I can’t tell into what that will bash into. I would think the command would be something like `docker exec -it hyrax bash`.